src/webui/webapplication.cpp

```diff
@@ -367,6 +367,8 @@ void WebApplication::action_command_download()
     CHECK_URI(0);
     QString urls = request().posts["urls"];
     QStringList list = urls.split('\n');
+    bool skipChecking = request().posts["skip_checking"].trimmed() == "true";
+    bool addPaused = request().posts["paused"].trimmed() == "true";
```

Comment:
OK. Then leave it as is. The WebUI code gives me headaches. I cannot follow fully throught the various classes. There might be a step that converts to all lowercase before it assignes the values to request().
Others will know. Just wait a few days.